pymc/distributions/discrete.py

```diff
@@ -220,7 +220,7 @@ def BetaBinom(a, b, n, x):
     ========  =================================================================
     Support   :math:`x \in \{0, 1, \ldots, n\}`
     Mean      :math:`n \dfrac{\alpha}{\alpha + \beta}`
-    Variance  :math:`n \dfrac{\alpha \beta}{(\alpha+\beta)^2 (\alpha+\beta+1)}`
+    Variance  :math:`\dfrac{n \alpha \beta (\alpha+\beta+n)}{(\alpha+\beta)^2 (\alpha+\beta+1)}
```

Comment:
The formula needs a closing `` ` `` in order to be rendered properly. Other than that it looks good, thanks!

Here is the preview for this PR: https://pymcio--6498.org.readthedocs.build/projects/docs/en/6498/api/distributions/generated/pymc.BetaBinomial.html